tests/pages/jobs/JobDetails-cy.js

```diff
@@ -9,28 +9,6 @@ xdescribe('Job Details', function () {
     cy.visitUrl({url: '/jobs/foo'});
   });
 
-  context('Job Details Header', function () {
-
-    it('renders the proper job name', function () {
```

Comment:
We don't want this test because it doesn't actually render the job's "full name" anymore, it renders the job's ID instead.